### PR TITLE
Orgin/feature/194 create separate nhais adaptor repository

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

File: nhais/nhais-env-example.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+NHAIS_OUTBOUND_RETRY_DELAY: "3"
+
+#User-specific env variables
+NHAIS_OUTBOUND_QUEUE_BROKERS: amqp://localhost:5672
+NHAIS_OUTBOUND_QUEUE_NAME: 'nhais_outbound'
+
+#common tests
+AWS_ACCESS_KEY_ID: test
+AWS_SECRET_ACCESS_KEY: test
+MHS_DYNAMODB_ENDPOINT_URL: http://localhost:8000
```
Comment (petervdm) [inline nhais/nhais-env-example.yaml]: Can you change the name to something NHAIS related?

File: nhais/sequence/tests/component_test_dynamo_sequence.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+from utilities import config
+from utilities import test_utilities
+
+from sequence import dynamo_sequence
+
+
+class ComponentTestDynamoSequence(unittest.TestCase):
+
+    def setUp(self):
+        config.setup_config("MHS")
```
Comment (petervdm) [inline nhais/sequence/tests/component_test_dynamo_sequence.py]: NHAIS

File: nhais/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Select File → New → Module from existing sources
+
+Point to NHAIS folder
+
+Select “Create module from existing sources"
+
+Click through wizard and select correct pipenv environment for NHAIS if it asks for one
+
+Open File → Project Structure… → select Modules → add root common (all_common) to NHAIS.
```
Comment (petervdm) [inline nhais/README.md]: Is this still required?

**Review by petervdm — CHANGES_REQUESTED** [review]

**Review by jaff47 — DISMISSED** [review]
Looks good

File: nhais/README.md (hunk 3 above)
Comment (Broom94) [reply nhais/README.md]: Good spot, it is now removed

File: nhais/sequence/tests/component_test_dynamo_sequence.py (hunk 2 above)
Comment (Broom94) [reply nhais/sequence/tests/component_test_dynamo_sequence.py]: changed

File: nhais/nhais-env-example.yaml (hunk 1 above)
Comment (Broom94) [reply nhais/nhais-env-example.yaml]: changed

**Review by petervdm — APPROVED** [review]

**Pull request merged**